Tests/Runtime/Meshes/OutOfPlaceMesh.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using JetBrains.Annotations;
+using UnityEngine;
+
+namespace TestHelper.Monkey.Meshes
+{
+    /// <summary>
+    /// Create a mesh that not contains a Transform position of the GameObject that the component attached to.
+    /// This mesh is able to hit raycast. Without no position annotations, monkey operators cannot operate the mesh
+    /// because a point where the monkey operators operate on is not in the mesh
+    /// </summary>
+    [RequireComponent(typeof(MeshCollider))]
+    [RequireComponent(typeof(MeshFilter))]
+    public class OutOfPlaceMesh : MonoBehaviour
```

Comment:
Add attribute to component in tests
```
[AddComponentMenu("")] // Hide from "Add Component" picker
```

Reply:
Fixed at [620e278](https://github.com/nowsprinting/test-helper.monkey/pull/47/commits/620e278e323668567aa4bb7d25ee1fd886a7df7d)